### PR TITLE
hashing in c++

### PR DESCRIPTION
**Pull request by VINAYBOYAT11**

### CODE REVIEW

**Review by jeremie1112 — CHANGES_REQUESTED** [review]
Hello,
Please read contribution.md and add the program in the respective path.
After that i'll accept your PR

**Comment by VINAYBOYAT11** [comment]
[@jeremie1112] check out my pull request

File: Data_structures/Hashing/c++/all_hashing_oprations.c++ [hunk 1]
```diff
@@ -0,0 +1,366 @@
+*/Hopefully, you’ve understood how a Hash Table can be implemented from scratch in C/C++ and possibly implemented one yourself!
+
+I would also suggest you try other collision handling algorithms and different hash functions on this Hash Table and test their performance./*
```
Comment (VINAYBOYAT11) [inline Data_structures/Hashing/c++/all_hashing_oprations.c++:3]: [comment box]

**Review by jeremie1112 — CHANGES_REQUESTED** [review]
Hello,
Can I ask you if this code was written by you ?
Also, the file name is not correct. It should be .cpp and not .c++

**Comment by VINAYBOYAT11** [comment]
no , am here just  contrubuting this code, it might help others 
and sorry  for that mistake ,i accendentily mistyped

**Comment by VINAYBOYAT11** [comment]
no ,but i understand this code and that why i contrubuting this code, it might help others
and sorry for that mistake ,i accendentily mistyped

**Review by jeremie1112 — APPROVED** [review]
Thanks for changes.
It's good now

**Comment by jeremie1112** [comment]
I understand, I really hope you understand this code and thank you for sharing

**Comment by VINAYBOYAT11** [comment]
ayy @jeremie1112   this is my first hactoberfest so 
thankyou sooo much ,for accepting my code and mentorship also..
am glad to meet you .

**Pull request merged**